manim/animation/creation.py

```diff
@@ -485,25 +485,30 @@ def update_submobject_list(self, index: int) -> None:
 
 
 # TODO, this is broken...
-class AddTextWordByWord(Succession):
+class AddTextWordByWord(ShowIncreasingSubsets):
     """Show a :class:`~.Text` word by word on the scene. Note: currently broken."""
 
     def __init__(
-        self,
-        text_mobject: "Text",
-        run_time: float = None,
-        time_per_char: float = 0.06,
-        **kwargs,
+            self,
+            text: "Text",
```

Comment:
Does this not work with `Tex` and `MathTex` then?

Reply:
I suppose not, but neither does AddTextLetterByLetter.